implementations/rust/ockam/ockam_api/src/nodes/models/forwarder.rs

```diff
@@ -20,35 +16,42 @@ pub struct CreateForwarder<'a> {
     #[cfg(feature = "tag")]
     #[n(0)] tag: TypeTag<3386455>,
     /// Address to create forwarder at.
-    #[b(1)] pub(crate) address: MultiAddr,
+    #[n(1)] pub(crate) address: MultiAddr,
     /// Forwarder alias.
     #[b(2)] pub(crate) alias: Option<CowStr<'a>>,
     /// Forwarding service is at rust node.
     #[n(3)] pub(crate) at_rust_node: bool,
-    /// Authorised identities per project.
-    ///
-    /// The current handling of this request will only create a secure channel
-    /// for addresses which have a corresponding authorised identity configured.
-    #[n(4)] pub(crate) identities: HashMap<MultiAddr, IdentityIdentifier>,
-    #[n(5)] pub(crate) mode: CredentialExchangeMode
+    #[n(4)] pub(crate) cloud_addr: Option<MultiAddr>,
+    #[n(5)] pub(crate) authorized: Option<IdentityIdentifier>
 }
 
 impl<'a> CreateForwarder<'a> {
-    pub fn new(
+    pub fn at_project(address: MultiAddr, alias: Option<String>, cloud: MultiAddr) -> Self {
+        Self {
+            #[cfg(feature = "tag")]
+            tag: Default::default(),
+            address,
+            alias: alias.map(|s| s.into()),
+            at_rust_node: false,
+            cloud_addr: Some(cloud),
+            authorized: None,
```

Comment:
Why `at_project` forwarder doesn't have authorized value?

Reply:
It uses the project authority.